### PR TITLE
Merge changes from 0.2.0 into development - Closes #352

### LINKED ISSUE #352: Merge recent changes from SDKv4 branch
### Description

Merge recent changes from SDKv4 branch (0.2.0).

### Motivation

- A lot of bug fixes were added to the `0.2.0`
- Standard database approach

### Acceptance Criteria

- Changes are merged
- Conflicts are resolved
- Tests are passing

### PR DESCRIPTION
**Pull request by nagdahimanshu**
### What was the problem?
This PR resolves #352 

### How was it solved?
- [x] Changes from 0.2.0 are merged into development
- [x] All Conflicts resolved

### How was it tested?
Local + Jenkins

### CODE REVIEW

File: services/core/shared/core/helpers/accountSearch.js [hunk 1]
```diff
@@ -13,32 +13,34 @@
  * Removal or modification of this copyright notice is prohibited.
  *
  */
-const config = require('../../../config');
-const db = require('../../database')[config.db.collections.accounts.db || config.db.defaults.db];
+// const config = require('../../../config');
```
Comment (sameersubudhi) [inline services/core/shared/core/helpers/accountSearch.js:16]: Please remove the file.

**Review by sameersubudhi — APPROVED** [review]
Looks good apart from the following remark.

File: services/core/shared/core/helpers/accountSearch.js (hunk 1 above)
Comment (nagdahimanshu) [reply services/core/shared/core/helpers/accountSearch.js:16]: Given change already done in the PR: https://github.com/LiskHQ/lisk-service/pull/338

**Review by MichalTuleja — APPROVED** [review]
LGTM 👍

**Pull request merged**